src/BlockTypes/ProductButton.php

```diff
@@ -192,7 +192,7 @@ protected function render( $attributes, $content, $block ) {
 			return apply_filters(
 				'woocommerce_loop_add_to_cart_link',
 				strtr(
-					'<div class="wp-block-button wc-block-components-product-button {classes} {custom_classes}"
+					'<div data-wc-interactive=true class="wp-block-button wc-block-components-product-button {classes} {custom_classes}"
```

Comment:
It's a boolean attribute, you don't need it to be true.

```suggestion
					'<div data-wc-interactive class="wp-block-button wc-block-components-product-button {classes} {custom_classes}"
```

Reply:
Good catch! Made the changes in 088604d, thanks 🙌🏻